lambdaclass/curse_of_myrra
# Implement player states icons And Uma marks

This PR implements
- Uma marks
- states icons (poison, disarmed, slowed , stunned)

To test it:

1. Start a game with someone else
2. Pick Uma, attack the other player with basicSkill , skill1 and skill2
3. 3 marks should appear in top of the character
4. The other player should see a poison icon on top of the screen

- [x] Tested in Unity Desktop
